File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        
        int size=nums.size();
        
        int i=0;
        
        vector<int>::iterator ptr;
        ptr = nums.begin()+1;
        
        while ( ptr < nums.end())
        {
            if(*ptr != *(ptr-1)) ptr++;
            else
                nums.erase(ptr);
        
        }

        
        return nums.size();
        
        
        
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] arr) {
        int i=0,j=0,n=arr.length;
        for( j=0;j<n;j++)
        {
            if(arr[i]!=arr[j])
            {
                i++;
                arr[i] = arr[j];
            }
        }
        return i+1;
    }
}